lib/Models/GltfCatalogItem.ts

```diff
@@ -99,19 +101,27 @@ export default class GltfCatalogItem
     }
     this._model.show = this.show;
 
-    let position;
-    if (this.origin !== undefined) {
+    let position: Cartesian3;
+    if (
+      this.origin !== undefined &&
+      this.origin.longitude !== undefined &&
+      this.origin.latitude !== undefined &&
+      this.origin.height !== undefined
```

Comment:
I thought about this, but the default should probably be to place it on terrain rather than the surface of the ellipsoid. This was just a very quick change to get geo-referenced gltfs working, so I just left it at this. Better to not have a default for now if we're going to change it later.